browser-switch/src/main/java/com/braintreepayments/api/BrowserSwitchClient.java

```diff
@@ -83,21 +81,15 @@ public BrowserSwitchClient(@NonNull Activity activity) {
     }
 
     /**
-     * Initialize the Auth Tab launcher. This should be called in the activity's onCreate()
-     * before the activity is started.
+     * Initialize the Auth Tab launcher. This should be called in the activity/fragment's onCreate()
+     * before it is started.
      *
-     * @param activity The activity used to initialize the Auth Tab launcher
+     * @param caller The ActivityResultCaller (Activity or Fragment) used to initialize the Auth Tab launcher
      */
-    public void initializeAuthTabLauncher(@NonNull Activity activity) {
```

Comment:
Now that this is called in the constructor, what are your thoughts on making this method private?